projects/pillow/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+cd Pillow
+make || true
```

Comment:
Why the `|| true`? Don't we want to know if/when the build is broken?

Reply:
The reason it was there was because the make script ran tests as well, which caused an issue since `libclang_rt*.so` was not preloaded. However, I got a fix now to do things cleanly.